src/js/schemas/job-schema/General.js

```diff
@@ -1,30 +1,30 @@
 /* eslint-disable no-unused-vars */
 import React from "react";
 /* eslint-enable no-unused-vars */
+import { i18nMark } from "@lingui/react";
 import JobValidatorUtil from "../../utils/JobValidatorUtil";
 import ValidatorUtil from "../../utils/ValidatorUtil";
 import MesosConstants from "../../../../plugins/services/src/js/constants/MesosConstants";
 import JobResources from "../../constants/JobResources";
 
 const General = {
-  title: "General",
-  description: "Configure your job settings",
+  title: i18nMark("General"),
```

Comment:
`i18nMark` does not actually translate anything - it just adds the marked text to the translation catalog. To actually translate the text, it has to be wrapped in `<Trans id={title} />` wherever it's used (where `title` would be whatever the marked variable is called where it's used)

Reply:
Thank you.